src/main/java/com/google/netpcapanalysis/dao/KeystoreDaoImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.google.netpcapanalysis.dao;
+
+import com.google.gson.Gson;
+import com.google.netpcapanalysis.interfaces.dao.KeystoreDao;
+import com.google.netpcapanalysis.models.Keystore;
+import java.io.FileReader;
+
+public class KeystoreDaoImpl implements KeystoreDao {
+
+  public static final String KEYSTORE_LOCATION = "keystore.json";
```

Comment:
Is this a single file for all keys or will there be multiple files?
You could store them in different files that will make it easier to remove, alternatively you could have a single entity where you store it as individual rows.